ui/apps/platform/src/services/AuthService/AuthService.ts

```diff
@@ -183,6 +172,27 @@ export function deleteAuthProviders(authProviderIds) {
     return Promise.all(authProviderIds.map((id) => deleteAuthProvider(id)));
 }
 
+function convertAuthProviderClaimMappingsToArray(provider: AuthProvider): AuthProvider {
+    if (!provider.claimMappings) {
+        return provider;
+    }
+    const mappingAsArray = Object.entries(provider.claimMappings).sort((a, b) =>
+        a[0].localeCompare(b[0])
```

Comment:
In theory `localeCompare` might have different sort order for different users, but probably is consistent for characters in keys.